Services/Interfaces/web_interface/static/js/components/configuration.js

```diff
@@ -613,6 +626,36 @@ function handle_activation_configuration_editor(){
     });
 }
 
+
+function handle_import_currencies(){
```

Comment:
What's the idea behing importing and exporting currencies instead of making another profile?

Reply:
import a currencies list in another profile/bot without creating another. Share it with other people.